technical-report/README.adoc

```diff
@@ -0,0 +1,23 @@
+= ISO International Standard template in Metanorma
```

Comment:
ISO Technical Report template in Metanorma

Reply:
Done